docs/high-avaliability/prometheus-HA.md

```diff
@@ -1,12 +1,62 @@
+# Running Promscale itself in High-Availability mode
+
+Promscale is a stateless service, thus it can safely be run with multiple
+replicas. A load-balancer can simply route Promscale requests to any replica.
+
 # Using Promscale with Prometheus in High Availability
```

Comment:
I don't think this is required any further. We can remove this and change the tense in the below paragraph to match the sense with the first para.

Reply:
I like this separation